src/components/menu/menu.jsx

```diff
@@ -0,0 +1,15 @@
+import ShowAllTasks from "./showAll";
+import HideCompletedTasks from "./hideCompleted";
+import DeleteCompletedTasks from "./deleteCompleted";
+
+const Menu = ({ showAllTasks, removeCompletedTasks, hideCompleted }) => {
+    return (
+    <>
+      <HideCompletedTasks hideCompleted={hideCompleted} />
+      <DeleteCompletedTasks removeCompletedTasks={removeCompletedTasks} />
+      <ShowAllTasks showAllTasks={showAllTasks} />
```

Comment:
each button should hold its logic an not get it from outside

Reply:
Ok